iphone/TitaniumKit/TitaniumKit/Sources/API/TiViewController.m

```diff
@@ -167,6 +168,31 @@ - (void)viewDidDisappear:(BOOL)animated
   [super viewDidDisappear:animated];
 }
 
+- (UIModalPresentationStyle)adaptivePresentationStyleForPresentationController:(UIPresentationController *)controller
+{
+  if ([_proxy conformsToProtocol:@protocol(TiWindowProtocol)]) {
+    UIModalPresentationStyle style = [(id<TiWindowProtocol>)_proxy adaptivePresentationStyleForPresentationController:controller];
+    return style;
+  }
+  return UIModalPresentationNone;
```

Comment:
The default should be `UIModalPresentationAutomatic` for iOS 13+ and `UIModalPresentationNone` for < iOS 13 as per [documentation](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621355-modalpresentationstyle?language=objc).